lib/get/get-full-source-space.js

```diff
@@ -94,5 +94,5 @@ function extractItems (response) {
 }
 
 function filterDrafts (items, includeDrafts) {
-  return includeDrafts ? items : items.filter((item) => !item.isPublished())
+  return includeDrafts ? items : items.filter((item) => !item.sys.publishedVersion)
```

Comment:
But this function is used to remove the drafts from the array, in case includeDrafts is false.
https://github.com/contentful/contentful-batch-libs/blob/54b0fcbd0da0d30426f9f2b17648aa142793edb1/lib/get/get-full-source-space.js#L28

So we should kick out all items that have no `publishedVersion`, not the ones that have it. Otherwise it would just import drafts and remove the published entries.